### ISSUE TITLE
In the fixed-height datagrid, switch users cannot scroll up/down using keyboard commands or MacOS Switch's scroll feature

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4612]
[Clarity Version]
3.1.1

[Test URL]
https://clarity.design/documentation/datagrid/fixed-height

2.1.1 Keyboard 
(Level A)
https://www.w3.org/TR/WCAG21/#keyboard

[OS]
MacOS 10.15.3, Windows 10

[User Agents/Assistive Technologies]
Safari 13.0.5, Firefox 75.0, Switch

[Component]
Datagrid

[Description]
In the fixed-height datagrid, switch users cannot scroll up/down using keyboard commands or MacOS Switch&#039;s scroll feature.

[User Impact]
Switch users cannot access data that is offscreen in the fixed-height datagrid.

[Steps to Reproduce]
See the following instructions for switch usage then follow the steps below.

Windows (see Scan through keys)
https://support.microsoft.com/en-us/help/10762/windows-use-on-screen-keyboard

Mac
https://support.apple.com/en-us/HT202865

1. Using a switch, navigate to the datagrid with the Tab key. 
2. Try scrolling up and down using the Up/Down arrow keys and notice that you cannot scroll the datagrid up/down.
For MacOS only: 
3. Try activating Pointer --&gt; Scroll Up or Scroll Down and notice that you cannot scroll the datagrid up/down.

[Recommendation]
No focusable elements exist within the Fixed-height datagrid. Without a focusable element inside the datagrid, Switch users and other users interacting with this content from the keyboard alone will be unable to scroll past the content that overflows the visible boundaries. Making the root grid element keyboard accessible with a tabindex value of &quot;0&quot; and managing focus with aria-activedescendant is one method of solving this issue.	&lt;div role=&quot;grid&quot; tabindex=&quot;0&quot; class=&quot;datagrid-table&quot; aria-activedescendant=&quot;[relevantCell]&quot;&gt;...&lt;/div&gt;

[Reporter Email] 
daman@damanw.com

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
